src/streaming-client.ts

```diff
@@ -104,6 +104,9 @@ export class RevAiStreamingClient extends EventEmitter {
             if (config.transcriber !== null && config.transcriber !== undefined) {
                 url += `&transcriber=${encodeURIComponent(config.transcriber)}`;
             }
+            if (config.language !== null && config.language !== undefined) {
```

Comment:
```suggestion
            if (config.language) {
```

Reply:
Do you think we should update all of the conditionals?

Reply:
this might be a techdebt, so I approved this PR.
if it is non-trivial change, let's have a separate tech debt story

Reply:
It doesn't work for the numerical parameters such as delete_after_seconds because `0` is a `falsey` value.

Reply:
it works for language and everything what is not `"0"`

Reply:
Yes. I have updated the conditionals to reflect this. Please check again.

Reply:
looks great